### PR TITLE
twitter stuff removed

### LINKED ISSUE #279: Remove Twitter auth option
Twitter is no longer supported as an auth option due to Twitter API changes. The auth button and corresponding code needs to be removed.

### PR DESCRIPTION
**Pull request by dfols**
Went through and removed all twitter stuff i could find!

closes #279

### CODE REVIEW

File: layouts/minimal.vue [hunk 1]
```diff
@@ -67,9 +67,6 @@ onMounted(() => {
                 <CoreLink to="https://github.com/davidmyersdev/octo" class="hover:underline">
                   GitHub
                 </CoreLink>
-                <CoreLink to="https://twitter.com/octowiki" class="hover:underline">
-                  Twitter
-                </CoreLink>
```
Comment (davidmyersdev) [inline layouts/minimal.vue:72]: This is a link to the Twitter for Octo, so it shouldn't be deleted. 😄

File: nuxt.config.ts [hunk 2]
```diff
@@ -46,12 +46,6 @@ export default defineNuxtConfig({
         { property: 'og:title', content: 'Octo | Write meaningful notes' },
         { property: 'og:description', content: 'Unlock the full potential of your accumulated knowledge.' },
         { property: 'og:image', content: 'https://octo.app/img/og.png' },
-        { name: 'twitter:card', content: 'summary_large_image' },
-        { property: 'twitter:domain', content: 'octo.app' },
-        { property: 'twitter:url', content: 'https://octo.app' },
-        { name: 'twitter:title', content: 'Octo | Write meaningful notes' },
-        { name: 'twitter:description', content: 'Unlock the full potential of your accumulated knowledge.' },
-        { name: 'twitter:image', content: 'https://octo.app/img/og.png' },
```
Comment (davidmyersdev) [inline nuxt.config.ts:54]: This is needed so that links to Octo show the correct preview in Tweets.

**Review by davidmyersdev — COMMENTED** [review]
Thanks for the PR! I noticed a couple of things that were deleted that we'll want to keep, but everything else looks great!

File: nuxt.config.ts (hunk 2 above)
Comment (dfols) [reply nuxt.config.ts:54]: heard 👍 let me fix that up

**Review by davidmyersdev — APPROVED** [review]

**Pull request merged**